react-native-purchases-ui/android/src/main/java/com/revenuecat/purchases/react/ui/views/WrappedCustomerCenterView.kt

```diff
@@ -0,0 +1,33 @@
+package com.revenuecat.purchases.react.ui.views
+
+import android.content.Context
+import android.util.AttributeSet
+import android.view.View
+import com.revenuecat.purchases.ui.revenuecatui.views.CustomerCenterView
+
+class WrappedCustomerCenterView(context: Context) : ComposeViewWrapper<CustomerCenterView>(context) {
+
+    override fun createWrappedView(context: Context, attrs: AttributeSet?): CustomerCenterView {
+        return CustomerCenterView(context, attrs)
+    }
+
+    fun setDismissHandler(dismissHandler: (() -> Unit)?) {
+        wrappedView?.setDismissHandler(dismissHandler)
+    }
+
+    // Ensure the view re-measures properly after loading state changes
```

Comment:
I think this comment doesn't apply here. But let's keep the method. We can probably move it to `ComposeViewWrapper` as well? And remove it from here and the wrapped paywall